auslib/admin/views/base.py

```diff
@@ -45,27 +45,38 @@ def decorated(*args, **kwargs):
     return wrap
 
 
+def catchOutDataError(messages):
```

Comment:
Better to name this "catchOutdatedDataError", to match the actual Exception name.